File: Data Structures & Algorithms/counting-bits/submission-1.cpp
class Solution {
public:
    vector<int> countBits(int n) {
        vector<int> res;
        for(int i=0;i<=n;i++){
            int c=0, num=i;
            while(num>0){
                num&=(num-1);
                c++;
            }
            res.push_back(c);
        }
        return res;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        mapp = {}
        for i, num in enumerate(nums):
            mapp[num] = i
        for i, num in enumerate(nums):
            diff = target - num
            if diff in nums and mapp[diff]!=i:
                return [i, mapp[diff]]